hydrogym/core.py

```diff
@@ -391,6 +489,7 @@ def reset(self, t=0.0) -> Union[ArrayLike, Tuple[ArrayLike, dict]]:
     self.flow.reset(q0=self.q0, t=t)
     self.solver.reset()
 
+    # Previously: return self.flow.get_observations(), info
```

Comment:
`master` still return `self.flow.get_observations()` only too.
(Linked to the question of the transition to Gymnasium )